src/sumologic-span-processor/index.spec.ts

```diff
@@ -179,10 +181,11 @@ describe('SumoLogicSpanProcessor', () => {
       childSpan.end(endTime); // end time is before page was hidden
       spanProcessor.onEnd(childSpan);
       jest.runAllTimers();
+      expect(span.attributes['document.visibilityState']).toBe('hidden');
```

Comment:
Why do we have `visibilityState` in child spans as well and why is it expected for the child to be `visible` in the line below?

Reply:
`document.visibilityState` is adding to all spans and tells us whether the page was visible by the whole span execution time.
In this test we do:
1. start with visible page
2. create root span and child span
3. save current time
4. change page to hidden
5. stop root span
6. stop child span with custom, previously saved time

The root span is `hidden`, because it ends when page is already hidden.
The child span is `visible`, because the page was visible by the child span ends time.